src/controller/python/README.md

```diff
@@ -220,7 +220,7 @@ ProductID: 0
 RequiresCustomFlow: 0
 RendezvousInformation: 0
 Discriminator: 3840
-SetUpPINCode: 12345678
+SetUpPINCode: 20202021
```

Comment:
If you're changing the PIN code in the outcome, the input to the `setup-payload` command should also be updated.